### PR TITLE
update the clinical fields

### PR DESCRIPTION
**Pull request by lindaxiang**

### CODE REVIEW

File: schemas/variant_calling_schema.json [hunk 1]
```diff
@@ -3,12 +3,82 @@
...
+                  "type": "string",
+                  "pattern": "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$"
+                },
+                "analysis_type": {
+                  "type": "string",
+                  "const": "sequencing_alignment"
+                }
+              }
+            },
+            "minItems": 2,
```
Comment (b-f-chan) [inline schemas/variant_calling_schema.json]: No issue here, just a curious question - Why is the `inputs` array constrained to exactly a size of 2 items?  Is it because you will have exactly one tumour analysis and one normal analysis ?

File: schemas/variant_calling_schema.json [hunk 2]
```diff
@@ -3,12 +3,82 @@
   "schema": {
```
Comment (b-f-chan) [inline schemas/variant_calling_schema.json:3]: Just a general question to refresh my memory, where/how do we supply the Project ID?  I don't see it in the schema

Comment (b-f-chan) [reply schemas/variant_calling_schema.json:3]: @lindaxiang - Just want to confirm, did you get a chance to review (and possibly correct) the relationships between the entities that I originally constructed in the schema?  I see you left some of the things I originally had so just wanted to check.  Thanks!

Comment (b-f-chan) [reply schemas/variant_calling_schema.json:3]: Not critical, just a small cleanup thing, we could remove the references to `exposure` and `family_history` in `donor` since those entities won't be used in the EUCANCan model

File: example_payloads/variant_calling.payload.json [hunk 3]
```diff
@@ -24,11 +24,10 @@
     {
```
Comment (b-f-chan) [inline example_payloads/variant_calling.payload.json:24]: Just a general/curious question about this sample payload JSON --> Why is this file needed if @edsu7 is creating the actual payloads we will use?  Or is this just a reference for people who want to create payloads and need an example?

**Review by b-f-chan — CHANGES_REQUESTED** [review]
@lindaxiang - Looks good thank you for doing this work on my behalf --> I just left a few comments, mostly just questions to understand how this works, but nothing critical

File: schemas/variant_calling_schema.json (hunk 1 above)
Comment (lindaxiang) [reply schemas/variant_calling_schema.json]: This is a good question. Considering this schema can potentially be extended to serve as the schema for 
- `germline` variant calling results
- `somatic` tumour only variant calling results
- `somatic` tumour + normal variant calling results 
We can loose the size of array to be 1 or 2 items.

File: schemas/variant_calling_schema.json (hunk 2 above)
Comment (lindaxiang) [reply schemas/variant_calling_schema.json:3]: > Just a general question to refresh my memory, where/how do we supply the Project ID? I don't see it in the schema

project_id or study_id are included in the base schema, so is not defined here in Dynamic ones.

Comment (lindaxiang) [reply schemas/variant_calling_schema.json:3]: > Not critical, just a small cleanup thing, we could remove the references to `exposure` and `family_history` in `donor` since those entities won't be used in the EUCANCan model

`exposure` and `family_history` are not defined in the schema, but they are listed in the property enum. we can remove them completely.

File: example_payloads/variant_calling.payload.json (hunk 3 above)
Comment (lindaxiang) [reply example_payloads/variant_calling.payload.json:24]: Yes. Your understanding is absolutely correct. The payload under `example_payloads` is to give an example and a reference for people who want to create payloads. And also we can use it to validate with the schema for some testing purpose.

File: schemas/variant_calling_schema.json (hunk 2 above)
Comment (b-f-chan) [reply schemas/variant_calling_schema.json:3]: Ah ok it is only in the enum, then it has no harm.  However, I think just to be clean, let us remove it from there also, if you don't mind :)

**Review by edsu7 — APPROVED** [review]
LGTM!

**Review by b-f-chan — APPROVED** [review]
Looks good thanks for the changes!
